rules/scripting-user-agents.data

```diff
@@ -46,3 +46,7 @@ winhttp.winhttprequest
 # https://kong.github.io/unirest-java/
 # User-Agent: unirest-java/VERSION
 unirest-java/
+# http library
```

Comment:
```suggestion
# http library (deprecated since 2022)
```

Reply:
@theseion 
I'm sorry I didn't quite get what I am supposed to do. Could you please explain again? Thank you!